source/mainnet/net/installation/downloads.rst

```diff
@@ -220,15 +220,15 @@ For the system requirements to run a node, see :ref:`System requirements to run
 
             To learn how to run a node on Mac, see :ref:`Run and manage a node on macOS  <run-node-macos>`.
 
-   .. tab:: Testnet  v3.0.2
+   .. tab:: Testnet  v4.0.11
 
       .. tabs::
 
          .. tab:: Ubuntu
 
             To run a node on a server with Ubuntu, you need a Debian package.
 
-               - `Download the Testnet Debian package <https://distribution.testnet.concordium.com/deb/concordium-testnet-node_3.0.2_amd64.deb>`_
+               - `Download the Testnet Debian package <https://distribution.testnet.concordium.com/deb/concordium-testnet-node_4.0.11_amd64.deb>`_
```

Comment:
We should check the link before publishing. At present it does not work.

Reply:
This link works.